input.html

```diff
@@ -35,6 +35,10 @@ <h2>Create token & grant access</h2>
                     <label for="expires_in">Token expiry (in seconds): </label>
                     <input type="text" class="form-control" value="3600" name="expires_in" id="expires_in"/>
                 </div>
+                <div class="form-group">
+                    <label for="refresh_token_expires_in">Refresh Token expiry (in seconds): </label>
+                    <input type="text" class="form-control" value="7200" name="refresh_token_expires_in" id="refresh_token_expires_in"/>
```

Comment:
Hello, can this be `<input type="number" />` please?

Reply:
Sure, should I change the token expires_in too? Or should I just do it to the refresh_token_expires_in?

Reply:
Sure, that makes sense, thanks!

Reply:
Done!